community/lmdb/build

```diff
@@ -0,0 +1,7 @@
+#!/bin/sh -e
+
+cd libraries/liblmdb
+
+# shellcheck disable=2086
+make CC=${CC:-cc}
```

Comment:
CC="${CC:-cc}"

Reply:
You could also replace the cd with a `make -C libraries/liblmdb` if you want.

Reply:
Ah i forgot that splitting cc is required only if you're invoking it directly.

make -C is not portable right ? Does bmake support it ?

Reply:
It does, but it's fine by me either way.